### PR TITLE
feat: add migration to jcloud

### PR DESCRIPTION
**Pull request by azayz**

### CODE REVIEW

**Review by deepankarm — CHANGES_REQUESTED** [review]
https://github.com/jina-ai/now/blob/e2702669a7d48f4522629edd98e709712712563a/now/deployment/deployment.py#L9

:point_up: needs update as `name` and `env_file` are not supported anymore.

File: requirements.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ docker==5.0.3
 requests==2.27.1
```
Comment (joschkabraun) [inline requirements.txt:12]: why are we not using jina version 3.9.4 here

**Review by joschkabraun — APPROVED** [review]
LGTM!

File: requirements.txt (hunk 1 above)
Comment (azayz) [reply requirements.txt:12]: you're right I missed this

**Review by deepankarm — APPROVED** [review]
:+1:

**Pull request merged**